cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -389,6 +389,12 @@ func (o *Options) ApplyTo(config *apiserver.Config, kubeClient kubernetes.Interf
 	gardenerAPIServerConfig.OpenAPIV3Config.Info.Title = "Gardener"
 	gardenerAPIServerConfig.OpenAPIV3Config.Info.Version = gardenerVersion.GitVersion
 
+	// For backward-compatibility, we also have to keep serving the /openapi/v2 endpoint since terraform-provider-kubernetes does not support OpenAPI v3 yet.
+	// For more details, see https://github.com/hashicorp/terraform-provider-kubernetes/issues/2769.
```

Comment:
Do you want to add a `TODO` to remove this after 2026 or so? Should be enough time for everybody 😅

Reply:
Shouldn't we remove OpenAPI v2 schema when terraform-provider-kubernetes supports OpenAPI v3?